src/utils/connection_manager.py

```diff
@@ -1,25 +1,67 @@
-class Borg:
-    _shared_state = {}
+from crate import client
```

Comment:
you should leave this class unchanged, because as said the idea is to replace directly in setup, secondly, you would need to proceed, so to say, in parallel and support both crate and timescale, easy if you keep this unchanged, if you touch this, you would need to adjust also timescale code, since it uses this code